addons/fortify/CfgWeapons.hpp

```diff
@@ -0,0 +1,16 @@
+class CfgWeapons {
+    class InventoryItem_Base_F;
+    class ACE_ItemCore;
+
+    class ACE_FortifyPickaxe: ACE_ItemCore {
+        author = ECSTRING(common,ACETeam);
+        displayName = "Pickaxe";
+        descriptionShort = "";
+        model = "\A3\Structures_F\Items\Tools\Hammer_F.p3d";
+        picture = QPATHTOF(UI\blank_co.paa);
+        scope = 2;
+        class ItemInfo: InventoryItem_Base_F {
+            mass = 0;
```

Comment:
Why does the item have a mass of 0?

Reply:
There is no such thing as a hammer than can generate hescos and steel towers out of thin air.
The item just represents the ability to do something, it isn't a real item.
I also don't want the item to not fit in a unit's loadout if it's added automatically.